MultiSource/Benchmarks/Olden/em3d/util.c

```diff
@@ -1,6 +1,6 @@
 /* For copyright information, see olden_v1.0/COPYRIGHT */
 
-#include <stdlib.h>
+#include <stdlib_checked.h>
```

Comment:
This I had missed before because none of the stdlib functions used use pointers, but i think it's best to actually do this change.